### PR TITLE
Sse 3181 accessibility tests product pages

### PR DESCRIPTION
**Pull request by raghu-kora**
https://govukverify.atlassian.net/browse/SSE-3181

Added accessibility tests for all product pages as defined [here](https://govukverify.atlassian.net/wiki/spaces/DFA/pages/3961946735/Product+site+pages) 

Included rules wcag2.1 & 2.2

### CODE REVIEW

File: features/accessibilityFeatures/accessibility.feature [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Feature: All product pages must meet the accessibility standards
+  Government services are legally required to be accessible.
+  This means services must comply with the international WCAG 2.1 AA
+  accessibility standard. This requirement has applied to all new and
+  existing public sector websites since 2020, and to native applications since June 2021.
+  The Government Digital Service (GDS) is working on how to assess the new success criteria in WCAG 2.2 and will start
+  monitoring for the extra criteria in October 2024. Until October 2024 we will monitor accessibility of websites and
+  apps to WCAG 2.1 level AA.
```
Comment (BorisP-GDS) [inline features/accessibilityFeatures/accessibility.feature:8]: comments from line 2 to line 8 not needed

File: features/accessibilityFeatures/accessibility.feature [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      | Contact us (get started)                                                     | /contact-us-details                                      |
+      | Contact us form                                                              | /contact-us                                              |
+      | Accessibility                                                                | /accessibility                                           |
+      | Privacy                                                                      | privacy-policy                                           |
+      | Cookies                                                                      | /cookies                                                 |
+      | Register to get started with GOV.UK One Login                                | /register                                                |
+      | Contact us                                                                   | /contact-us                                              |
+
+
+
```
Comment (BorisP-GDS) [inline features/accessibilityFeatures/accessibility.feature:44]: Remove empty lines 43-44

File: package.json [hunk 3]
```diff
@@ -16,8 +16,9 @@
     "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' nyc mocha --require ts-node/register --ui bdd 'test/**/*.spec.ts'",
     "test:cov": "nyc --reporter=lcov npm run test --reporter mocha-sonarqube-reporter --exit",
     "sonar": "sonar-scanner",
-    "cucumber": "cucumber-js --require-module ts-node/register --require features/support/test-setup.ts --require features/support/steps/**/*.ts --publish-quiet --tags 'not @wip'",
+    "cucumber": "cucumber-js --require-module ts-node/register --require features/support/test-setup.ts --require features/support/steps/**/*.ts --tags 'not @wip'",
```
Comment (BorisP-GDS) [inline package.json:19]: Why is --publish-quiet removed ?

**Review by BorisP-GDS — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (raghu-kora) [reply package.json:19]: `publishQuiet` option is no longer needed, you can remove it from your configuration; see https://github.com/cucumber/cucumber-js/blob/main/docs/deprecations.md

Keep seeing this warning in the console

**Pull request closed without merging**